components/removed_from_channel_modal/removed_from_channel_modal.jsx

```diff
@@ -0,0 +1,125 @@
+
+            /*
+             * Action creator to close modal
+             */
+            closeModal: PropTypes.func.isRequired,
+        }).isRequired,
+    }
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            show: true,
+        };
+    }
+
+    handleClose = () => {
+        this.setState({show: false});
+    }
+
+    render() {
+        const {currentUser, actions: {closeModal}} = this.props;
+
+        var channelName = (
+            <FormattedMessage
+                id='removed_channel.channelName'
+                defaultMessage='the channel'
+            />
+        );
+
+        if (this.props.channelName) {
+            channelName = this.props.channelName;
+        }
+
+        var remover = (
+            <FormattedMessage
+                id='removed_channel.someone'
+                defaultMessage='Someone'
+            />
+        );
+
+        if (this.props.remover) {
+            remover = this.props.remover;
+        }
+
+        if (currentUser != null) {
+            return (
+                <Modal
+                    show={this.state.show}
+                    onHide={this.handleClose}
+                    onExited={() => closeModal(ModalIdentifiers.REMOVED_FROM_CHANNEL)}
+                    id={ModalIdentifiers.REMOVED_FROM_CHANNEL}
+                >
+                    <Modal.Header closeButton={true}>
+                        <Modal.Title>
+                            <FormattedMessage
+                                id='removed_channel.from'
+                                defaultMessage='Removed from '
+                            />
+                            <span className='name'>{channelName}</span>
+                        </Modal.Title>
+                    </Modal.Header>
+                    <Modal.Body>
+                        <p>
+                            <FormattedMessage
+                                id='removed_channel.remover'
+                                defaultMessage='{remover} removed you from {channel}'
+                                values={{
+                                    remover,
+                                    channel: (channelName),
+                                }}
+                            />
+                        </p>
+                    </Modal.Body>
+                    <Modal.Footer>
+                        <button
+                            type='button'
+                            className='btn btn-primary'
+                            data-dismiss='modal'
+                            onClick={() => this.handleClose()}
+                        >
+                            <FormattedMessage
+                                id='removed_channel.okay'
+                                defaultMessage='Okay'
+                            />
+                        </button>
+                    </Modal.Footer>
+                </Modal>
+            );
+        }
+
+        return <div/>;
```

Comment:
Another minor nit, but this can be `return null`

Reply:
Since we're already returning `null` when the `currentUser` prop is null, I figure'd I'd just remove this line instead